Chore: update Readme.rst

xmagor opened the pull request:
Include Build process to use the wrapper CoolProp to Goland

### Description of the Change

Add the necessary process to build the CoolProp compile wrapper for goland

### Benefits

It would be easier and clearer for new contributors and for us to remember how to build the CoolProp container. 

### Possible Drawbacks

It could be that the workflow its doing to Ubuntu users only

GautierR committed:
Merge pull request #1 from GautierR/golang_module

Golang module

GautierR committed:
Adding the compiled library to the repository

GautierR committed:
Add go.mod

GautierR committed:
Sed build flags into CoolProp.go

GautierR committed:
Add libdl.a

GautierR committed:
Changing Go folder to go

GautierR committed:
Handle conflict

xmagor committed:
Chore: update Readme.rst

Include compile process to use the wrapper CoolProp to Go

jon85p commented:
How about modify the CMakeLists file for generate the headers file for Golang wrapper without using double compilation?

xmagor commented:
> 
> 
> How about modify the CMakeLists file for generate the headers file for Golang wrapper without using double compilation?

@jon85p  That was really fast "saying and doing"  dc13375 👍 

I'll do the pertinent changes in the Readme! 🔥

xmagor committed:
Chore: Readme you don't need to compile twice

Modify workflow build wrapper for goland due to now is not
necesary first build the original CoolProp to get gitrevision.h file

jon85p commented:
Nice!, but.. hmm
Pollute this PR with binaries? :eyes:
